Software/Rig/trial.h

```diff
@@ -15,8 +15,8 @@ const int NUMBER_OF_TRIALS = 6;
 // NOTE: this is only used for debugging, the "real" trial uses a
 // random value between 1min and 5min
 const long INTER_TRIAL_DEBUG_WAIT_INTERVAL = 1000; // ms
-const long MIN_ITI = 1000; // ms
-const long MAX_ITI = 5000; // ms
+const long MIN_ITI = 60000; // ms
+const long MAX_ITI = 300000; // ms
```

Comment:
NOTE: the shorter inter-trial intervals were set for testing. This pr updates them to their intended values as part of the commentary on maximum session length